be1-go/channel/lao/mod.go

```diff
@@ -711,6 +688,25 @@ func (c *Channel) processRollCallClose(msg messagedata.RollCallClose, socket soc
 	return nil
 }
 
+func verifySenderIsOrganizer(msg message.Message, org kyber.Point) error {
```

Comment:
this function could arguably be part of the Channel. Then you already have access to the organizerPukey and must only pass the message.